src/core/render/utils.js

```diff
@@ -23,8 +23,8 @@ export function getAndRemoveConfig(str = '') {
 
   if (str) {
     str = str
-      .replace(/^('|")/, '')
-      .replace(/('|")$/, '')
+      .replace(/^"(:.*)"$/, '$1')
+      .replace(/^'(:.*)'$/, '$1')
```

Comment:
It would be nice to have the matching-quotes addition without the `:` change. A separate PR for the `:` could be saved for v5 (breaking change).